migrations/20210529144922-initial-table-creation.js

```diff
@@ -24,6 +24,12 @@ module.exports = {
       },
       deletedAt: { type: Sequelize.DATE }
     })
+
+    await queryInterface.createTable('novels', {
+      id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+      title: { type: Sequelize.STRING, allowNull: false },
+      authorId: { type: Sequelize.INTEGER, references: { model: Authors, key: 'id' } },
```

Comment:
You forgot this on this table and your NovelsGenres

createdAt: { type: Sequelize.DATE, allowNull: false, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'), },      updatedAt: {        type: Sequelize.DATE,        allowNull: false,        defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'),      },      deletedAt: { type: Sequelize.DATE, },